pyvista/utilities/geometric_objects.py

```diff
@@ -768,6 +768,9 @@ def Polygon(center=(0.0, 0.0, 0.0), radius=1, normal=(0, 0, 1), n_sides=6):
     n_sides : int, optional
         Number of sides of the polygon.
 
+    generate_polygon : bool, optional
```

Comment:
If possible we should give a more meaningful name to this. Something like `fill` or similar. Anything "polygon" seems confusing in a callable called "Polygon". A better name would also make it clear which value of the kwarg leads to filling.

Reply:
I didn't notice `fill` naming. Thanks.
```suggestion
    fill : bool, optional
```